CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # MetovaBase CHANGELOG
 
+## 2.0.0
```

Comment:
Do you feel this deserves a major version bump?

Reply:
Probably not in my book. I'm good with 1.1

Reply:
This sounds good to me!